### PR TITLE
modify: help information

### PR DESCRIPTION
**Pull request by ycyaoxdu**
Signed-off-by: ycyaoxdu <yaoyuchen0626@163.com>

### CODE REVIEW

**Comment by ycyaoxdu** [comment]
```
root@yuchen clusteradm [refactor] $ clusteradm help
clusteradm controls the OCM control plane.

 Find more information at: https://github.com/open-cluster-management-io/clusteradm/blob/main/README.md

General commands:
  create      create a resource
  delete      delete a resource
  get         get information from the cluster
  install     install a feature
  upgrade     upgrade componenet
  version     get the versions of different components

Registration commands:
  accept      accept a list of clusters
  clean       clean the hub
  init        Initialize a Kubernetes cluster into an OCM hub cluster.
  join        join a cluster to the hub
  unjoin      unjoin from a hub

Customized Resource command:
  addon       addon options
  clusterset  clusterset options
  proxy       helper commands for cluster-proxy addon

Other Commands:
  completion  Generate the autocompletion script for the specified shell
  config      Modify kubeconfig files
  plugin      Provides utilities for interacting with plugins

Usage:
  clusteradm [flags] [options]

Use "clusteradm <command> --help" for more information about a given command.
Use "clusteradm options" for a list of global command-line options (applies to all commands).
```

File: cmd/clusteradm/clusteradm.go [hunk 1]
```diff
@@ -75,29 +82,39 @@ func main() {
...
-				deletecmd.NewCmd(clusteradmFlags, streams),
+				acceptclusters.NewCmd(clusteradmFlags, streams),
 				clean.NewCmd(clusteradmFlags, streams),
 				inithub.NewCmd(clusteradmFlags, streams),
 				joinhub.NewCmd(clusteradmFlags, streams),
 				unjoin.NewCmd(clusteradmFlags, streams),
+			},
+		},
+		{
+			Message: "Customized Resource command:",
```
Comment (qiujian16) [inline cmd/clusteradm/clusteradm.go:104]: that is a bit weird to say it is Customized Resource command

Comment (ycyaoxdu) [reply cmd/clusteradm/clusteradm.go]: how about `Cluster Management commands`?

Comment (qiujian16) [reply cmd/clusteradm/clusteradm.go]: yeah, that is better

**Comment by qiujian16** [comment]
@yue9944882 any thought on this?

**Review by yue9944882 — COMMENTED** [review]
/lgtm
/approve

**Pull request merged**